public class ExceptionJava {

	public static void main(String[] args) {

		/*for (int i = 0; i <= 9; i++) {
			try {
				System.out.println("try");
				if (i == 5) {
					i = i / 0;
					break;
				}
			} catch (ArithmeticException e) {
                System.out.println("catch");
			} finally {
				System.out.println("finally");
			}
		}*/
		
		/*for (int i = 0; i <= 9; i++) {
			try {
				System.out.println("try");
				if (i == 5) {
					i = i / 0;
					continue;
				}
			} catch (ArithmeticException e) {
                System.out.println("catch");
			} finally {
				System.out.println("finally");
			}
		}*/
		int i=m1();
		System.out.println(i);
	}
	public static int m1(){
			try {
				  System.out.println("try");
				   return 10/0;
				}
			catch (ClassCastException e) {
                System.out.println("catch");
                return 20;
			} finally {
				System.out.println("finally");
				return 70;
			}
		}
}
